aea/skills/base.py

```diff
@@ -43,7 +43,7 @@
 from aea.context.base import AgentContext
 from aea.contracts.base import Contract
 from aea.crypto.ledger_apis import LedgerApis
-from aea.decision_maker.base import GoalPursuitReadiness, OwnershipState, Preferences
+from aea.decision_maker.active import GoalPursuitReadiness, OwnershipState, Preferences
```

Comment:
note, here we load the implementation rather than base and then cast appropirately.

Reply:
see above